### DISCUSSION TITLE
War students

### DISCUSSION THREAD

**Discussion by Ahmed-Ibrahim-a-a** [community/community#65184] [GitHub Education] [answered]
Hi
I hope this message find you well
I have a problem with signing for a GitHub student pack
Currently i live in KSA because of the war in my country (Sudan) but i didn't finish university yet and i have certificates and university id and they both in arabic that can approve that, but when i try to sign up it tells me that iam far away from my collage campus or they think iam using vpn, and i don't know what else to do .if there is someone who can read Arabic and verify my certificates and help me get the students pack i will be thankful.

Anyway thanks for your time

**Comment by Machine1336** [comment]
Okay

**Reply by Ahmed-Ibrahim-a-a** [reply]
You can help?

**Reply by Machine1336** [reply, marked as answer]
What if you contacted GitHub to try and explain the situation.
I am sorry for what you are going through, but the only and best available solution is through contacting github support

**Reply by Ahmed-Ibrahim-a-a** [reply]
Oh ok thanks
